rhc-ose-ansible/roles/secure-registry/defaults/main.yaml

```diff
@@ -0,0 +1,13 @@
+openshift_registry_project: "default"
+docker_registry_dc_name: "docker-registry"
+registry_certificate: ""
+registry_key: ""
+registry_ca: "/etc/origin/master/ca.crt"
+registry_serviceaccount: "registry"
+registry-secret-name: "registry-secret"
+openshift:
+   common:
+     client_binary: "oc"
+     admin_binary: "oadm"
+openshift.common.admin_binary: "oadm"
+secure_registry_master_host: play_hosts[0]
```

Comment:
'play_hosts[0]' is interpreted as a literal string. Line should be:

`secure_registry_master_host: "{{ play_hosts[0] }}"`